src/extension.ts

```diff
@@ -416,6 +416,7 @@ export namespace Openshift {
         const loginMessage = async (clusterURL, result, explorer)=> {
             if (result.stderr === "") {
                 explorer.refresh();
+                vscode.commands.executeCommand('setContext', 'isLoggedIn', true);
```

Comment:
You set context only when you login/logout commands executed, but you are forgetting about initial state, right after extension activation.

Reply:
I have added the initial state scenario too in this:
https://github.com/redhat-developer/vscode-openshift-tools/pull/141/files#diff-f8d556788b6298850280fcb4c474b5e2R168